.gitignore

```diff
@@ -50,3 +50,9 @@ kratos/includes/kratos_version.h
 # Output of tests of meshing application (MMG)
 *.o.sol
 *.o.mesh
+
+cmake_build/
```

Comment:
I would like to add that cmake_build is already ignored anyway: https://github.com/KratosMultiphysics/Kratos/blob/62e58da2bf9b9a69ab4c64bf307fa37369bb92e2/.gitignore#L5